src/lib/components/Swap/SwapButton.tsx

```diff
@@ -153,6 +160,20 @@ export default function SwapButton({ disabled }: SwapButtonProps) {
       })
   }, [addTransaction, independentField, inputCurrencyAmount, outputCurrencyAmount, setDisplayTxHash, swapCallback])
 
+  if (wrapType !== WrapType.NOT_APPLICABLE) {
```

Comment:
The same Action button needs to be rendered regardless of wrap/swap in order for css transitions to function for the button. Instead of returning early, set the props to then be passed to the ActionButton.